src/main/java/com/bina/varsim/tools/simulation/VCF2diploid.java

```diff
@@ -361,7 +361,7 @@ private boolean addVariant(final byte[] maskedSequence, final Sequence reference
         variants in the same haplotype should never overlap.
         Otherwise, they should be merged at first place.
          */
-        for (int p = position; p < position + referenceAlleleLength; p++) {
+        for (int p = position; p <= position + referenceAlleleLength; p++) {
```

Comment:
Wouldn't this flag deletion side by side with an insertion as overlapping? An insertion only overlaps if it is flanked by deletion on both sides.